Source/Core/InputCommon/ControllerInterface/OSX/OSXJoystick.mm

```diff
@@ -109,8 +109,8 @@
         break;
       }
 
-      NOTICE_LOG(SERIALINTERFACE, "Unknown IOHIDElement, ignoring (Usage: %x, Type: %x)\n", usage,
-                 IOHIDElementGetType(e));
+      NOTICE_LOG_FMT(SERIALINTERFACE, "Unknown IOHIDElement, ignoring (Usage: {:x}, Type: {:x})\n",
```

Comment:
Do we need that `\n` at the end? There's nothing else in that message beyond it, and every message is its own line by default anyways.